src/app/static/src/app/components/Stepper.vue

```diff
@@ -7,6 +7,7 @@
                       :number="step.number"
                       :text="step.text"
                       :enabled="enabled(step.number)"
+                      :complete="status[step.number]"
```

Comment:
Might be a little less confusing to rename status to complete if it's just a complete/incomplete bool.